src/core/core.scale.js

```diff
@@ -331,6 +335,29 @@ var Scale = Element.extend({
 		return me.minSize;
 
 	},
+
+	/**
+	 * @private
+	 */
+	_configure: function() {
+		var me = this;
+		var reverse = me.options.ticks.reverse;
+		var start, end;
+
+		if (me.isHorizontal()) {
+			start = me.left;
+			end = me.right;
+		} else {
+			start = me.top;
```

Comment:
I would expect `start = me.bottom` and am also surprised by `reverse = !reverse`

Reply:
These are about pixels. So top/left is 0,0. Could name these `_startPixel` etc. `reverse` is also about pixels here.